moja-kuhinjica-frontend/src/components/menu/MenuItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import Image from 'next/image'
+import { ColDiv, RowDiv } from '@/styles/global'
+import pic from '../../../public/static/assets/images/meal1.png'
+import {
+    TitleWrapper,
+    TitleLabel,
+    PriceWrapper,
+    ContentLabel,
+    PriceLabel,
+    DescriptionLabel,
+} from './style'
+
+export const MenuItem = () => {
+    const [openDescription, setOpenDescription] = useState(false)
+
+    return (
+        <ColDiv width="420px" height="700px" alignItems="center">
+            <Image src={pic} alt="" />
+            <TitleWrapper height="20%" rotation={openDescription ? -90 : 90}>
+                <TitleLabel>Piletina u sosu od šampinjona</TitleLabel>
+                <svg
+                    onClick={() => setOpenDescription(!openDescription)}
```

Comment:
You should import this svg from svg file from assets. Not to inject it directly in code

Reply:
But this SVG is not static. In css I change fill color, do rotation too. Would I still be able to do it if I import svg from assets?

Reply:
Please then put this svg with rotation and fill in separate  component

Reply:
Done.